### PR TITLE
Gen NEXT: Descriptions

### PR DESCRIPTION
**Pull request by DeathlyPlays**

### CODE REVIEW

**Comment by panpawn** [comment]
... Don't open duplicate PRs. We literally told you to rebase, and if you needed help to ask. >_>

**Comment by DeathlyPlays** [comment]
I tried rebasing >_> I got it done then I couldn't push the rebased commit to that branch >_>

**Comment by panpawn** [comment]
You have to force push after rebasing. If you had asked us for help, then you'd of known that :|

**Comment by AustinXII** [comment]
Gotta agree with panpawn, he offered you help multiple times and you just disregarded it for no reason. You are at fault here

**Comment by DeathlyPlays** [comment]
~~Anyways can the comments on this PR be actually about what the PR is about?~~

**Comment by Zarel** [comment]
I swear I've told you not to do this before. We both offered to help you, explicitly because we didn't want you to do this. If you do this again I'm going to ban you from this repository; I've already banned someone else for repeatedly doing this sort of thing.

**Comment by Zarel** [comment]
Literally the last thing I said was "Tell me if you need more help."

**Comment by DeathlyPlays** [comment]
Feel free to ban me *sighs*

**Comment by Zarel** [comment]
I won't, this is just a warning...

**Comment by Zarel** [comment]
Seems like an improvement. Can always commit corrections if it turns out something's wrong.

**Comment by DeathlyPlays** [comment]
Cool. :p

File: mods/gennext/README.md [hunk 1]
```diff
@@ -145,7 +145,7 @@ Major changes:
 
 - Twister is now a 80 base power Flying move with a 30% confusion chance
 
-- Floette-Eternal-Flower and Volcanion are released.
+- Floette-Eternal-Flower are released.
```
Comment (urkerab) [inline mods/gennext/README.md]: Grammar nit: Floette-Eternal-Flower is released.

Comment (DeathlyPlays) [reply mods/gennext/README.md]: Ill fix this

Comment (Zarel) [reply mods/gennext/README.md]: ...I already fixed this

Comment (DeathlyPlays) [reply mods/gennext/README.md]: I noticed after replying

**Pull request merged**